components/search-results/download-tsv.js

```diff
@@ -0,0 +1,13 @@
+import { DocumentArrowDownIcon } from "@heroicons/react/24/outline";
+import { API_URL } from "../../lib/constants";
+import { ButtonLink } from "../form-elements";
+
```

Comment:
At the very least for consistency’s sake, could you break these up into sections like the other files and sort them alphabetically so that once these import lists get very long, it’s easy to find an import without having to do a search? Also describe what the `DownloadTSV` component is for like all the other components.